Workbooks/Intune Audit/Audit/IntuneAuditActivity.workbook

```diff
@@ -0,0 +1,390 @@
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 86400000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 172800000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 259200000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 604800000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 1209600000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 2592000000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 5184000000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 7776000000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                }
+              ],
+              "allowCustom": true
+            },
+            "timeContextFromParameter": null
+          }
+        ],
+        "style": "pills",
+        "resourceType": "microsoft.resourcegraph/resources"
+      },
+      "conditionalVisibility": null
+    },
+    
+    {
+      "type": 1,
+      "content": {
+        "json": "## Audit Events Trend"
+      },
+      "conditionalVisibility": null
+    },
+    
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "IntuneAuditLogs\r\n| summarize count() by bin(TimeGenerated, time(1h))",
```

Comment:
for the `bin` part, you probably want this to be something like `bin(TimeGenerated, {TimeRange:grain})` to inherit a default grain from the selected `TimeRange` parameter instead of always being 1 hour?  (from math I can't tell what the time ranges are, but using `{TimeRange:grain}` formatter will get you a reasonable bin grain from a time setting

Reply:
Thank you for the feedback. I updated the review with it.